Core/DISET/RequestHandler.py

```diff
@@ -78,6 +79,24 @@ def _rh__initializeClass( cls, serviceInfoDict, lockManager, msgBroker, monitor
     cls.__monitor = monitor
     cls.log = gLogger
 
+    allNames = dir( cls )
+    exportedMethods = [ name for name in allNames if 'export_' in name ]
+    system, service = cls.__svcName.split( '/' )
+    setup = gConfigurationData.extractOptionFromCFG( 'DIRAC/Setup' )
+    systemSetup = gConfigurationData.extractOptionFromCFG( 'DIRAC/Setups/%s/%s' % ( setup, system ) )
+    result = gConfigurationData.extractOptionFromCFG( '/Systems/%s/%s/Services/%s/ResponseMonitoring/Methods' % ( system, systemSetup, service ) )
```

Comment:
Split the line please.
Also, if you use gConfig.getOption with a default value of a list, it will by default split the content using the coma as separator